src/ComposerScripts.php

```diff
@@ -60,8 +60,14 @@ public static function postDrupalScaffold() {
     // Symlink the libraries folder into the Drupal core git repo.
     static::makeSymlink('../../web/libraries', 'repos/drupal/libraries');
 
-    // Symlink the contrib profiles folder into the Drupal core git repo.
-    static::makeSymlink('../../../web/profiles/contrib', 'repos/drupal/profiles/contrib');
+    // Symlink external and custom folders into the Drupal core git repo.
```

Comment:
That's not the way this is supposed to work.

Drupal needs to find these things in the actual app root.

Reply:
That works well enough for Drupal, but Drush won't work unless DRUPAL_ROOT is set correctly.

Reply:
I thought Drush was now getting the app root from DrupalKernel? Whereabouts is Drush using DRUPAL_ROOT?

Reply:
I was mistaken. Drush was throwing an error because a contrib module relied on DRUPAL_ROOT, not Drush itself. This whole pull request is about preventing errors caused by DRUPAL_ROOT being repos/drupal instead of the app root when Drupal is installed with a symlink. That issue is not likely to be fixed in current versions of Drupal. This simple fix makes them work.

Reply:
This problem has already been recognized. That's why we symlink the vendor and simpletest folders. This just completes that solution.